cluster-scope/overlays/nerc-ocp-test/machineconfigs/udev-rules/src/bcm57412.rules

```diff
@@ -0,0 +1,2 @@
+SUBSYSTEM=="net",DRIVERS=="bnxt_en",KERNELS=="0000:18:00.0",ACTION=="add",NAME="nic1"
+SUBSYSTEM=="net",DRIVERS=="bnxt_en",KERNELS=="0000:b3:00.1",ACTION=="add",NAME="nic2"
```

Comment:
I need to check the value of `KERNELS` on the second line to make sure that difference in the second position isn't a copy/paste error.

Reply:
Fixed; that was a typo.